LocalSettings.php

```diff
@@ -1772,6 +1767,16 @@
 			'Varnish' => 'https://meta.miraheze.org/wiki/Tech:Varnish',
 		],
 	],
+	'wgIncidentReportingInactiveServices' => [
```

Comment:
These services should stay in the other config, and this array should just include the names, not the URLs as well (at least for now in the current state of my PR, but if wanted it can be changed to be done like this)

Reply:
I might change the PR to be like this though instead.

Reply:
I tried that initially and they stayed in the selectable list.